### DISCUSSION TITLE
Favicon not showing up

### DISCUSSION THREAD

**Discussion by crajnc** [vercel/community#2753] [Help] [answered]
### Summary

Favicon not showing up, we tried with the default method like placing our Favion in the public folder & also specifically mentioning it in the `<head/>` element as well. 

Have also Purged the Data Cache as well.

Please help 🙏

### Example

https://newcycl.com/

### Steps to Reproduce

1. Visit this link - [https://newcycl.com](https://newcycl.com), but the Favicon is working in [https://newcycl.in](https://newcycl.in)

**Comment by swarnava** [comment]
Hi,

I can see that favicon is present:

<img width="462" alt="image" src="https://github.com/vercel/community/assets/1162991/48b91d68-e6cc-42e1-8714-a49ed76ca64c">

Are you still seeing this behavior?

**Reply by crajnc** [reply]
can you check here - https://newcycl.com/ it's not showing up here..

**Reply by swarnava** [reply, marked as answer]
I can see it there as well 

<img width="595" alt="image" src="https://github.com/vercel/community/assets/1162991/7a28479f-bc3f-4ed6-ba33-5926c8b8d9dd">
